src/tpt_reports/tpt_reports.py

```diff
@@ -1,20 +1,21 @@
 """cisagov/tpt-reports: A tool for creating Technical Phishing Test (TPT) reports.
 
 Usage:
-  tpt-reports ASSESSMENT_ID ELECTION_NAME DOMAIN_TESTED JSON_FILE_PATH OUTPUT_DIRECTORY [--log-level=LEVEL]
+  tpt-reports ASSESSMENT_ID ELECTION_NAME DOMAIN_TESTED JSON_FILE_PATH [--output-dir=OUTPUT_DIRECTORY] [--log-level=LEVEL]
```

Comment:
Might as well keep the optional arguments in alphabetical order, same as listed in the "Options" section below:
```suggestion
  tpt-reports ASSESSMENT_ID ELECTION_NAME DOMAIN_TESTED JSON_FILE_PATH [--log-level=LEVEL] [--output-dir=OUTPUT_DIRECTORY]
```